react/features/base/util/getUnsafeRoomText.web.ts

```diff
@@ -0,0 +1,22 @@
+import { translateToHTML } from '../i18n/functions';
+
+/**
+ * Gets the unsafe room text for the given context.
+ *
+ * @param {Function} t - The translation function.
+ * @param {'meeting'|'prejoin'|'welcome'} context - The given context of the warining.
+ * @returns {string}
+ */
+export default function getUnsafeRoomText(t: Function, context: 'meeting' | 'prejoin' | 'welcome') {
+    const defaultSecurityUrl = 'https://jitsi.org/security/';
+    const securityUrl = typeof APP === 'undefined'
```

Comment:
You don't need to do this if you are in a .web file, APP is guaranteed to exist.